kylin-spark-project/kylin-spark-engine/src/main/scala/org/apache/spark/sql/KylinSparkEnv.scala

```diff
@@ -25,96 +25,27 @@ object KylinSparkEnv extends Logging {
 	@volatile
 	private var spark: SparkSession = _
```

Comment:
Are you plan to delete `KylinSparkEnv` ?

Reply:
Yeah, will raise another pr to do this, it needs to refactor some modules